src/PhpSpreadsheet/Style/Font.php

```diff
@@ -255,7 +255,7 @@ public function getSize()
      */
     public function setSize($pValue)
     {
-        if ($pValue == '') {
+        if ($pValue == '' || $pValue == 0) {
```

Comment:
I think the problems with invalid values lie a little bit deeper than simply trying to set a font size to 0; try setting it to -10 and see what the resulting value is... if you're going to fix the method when given an invalid value, you should probably fix it for any invalid value, not just 0